Sem. 03/Pract.03/Task.12.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include <iostream>
+using namespace std;
+
+
+int main()
+{
+	int N;
+	cin >> N;
+	int num;
+	int sum = 0;
+	while (true) {

```

Comment:
Rather than looping infinitely on line 12, you could rewrite the while loop by getting the first num before the loop, then the while loop would be making a check like so: while (num != -1). Then, at the end of the while loop, you would be making std::cin on a new number